docs/websocket/search.md

```diff
@@ -0,0 +1,106 @@
+# Search
+
+Search for playlists or tracks based on a query.
+
+## Request
+
+```json
+{
+    "id": 7,
+    "type": "spotcast/search",
+    "query": "rock",
+    "searchType": "playlist",
+    "limit": 10,
+    "account": "01JDG07KSBTYWZGJSBJ1EW6XEF"
+}
+```
+### `id` (int)
+
+The id of the transaction. Must be an increment of the last transaction id.
+
+### `type` (str)
+
+The endpoint of the WebSocket to reach. Must be `spotcast/search`.
+
+### `query` (str)
+
+The search query string. This can be any term, such as a track name, album, artist, playlist name, or genre.
+
+### `searchType` (str)
+
+*Optional*
+
+The type of search. Can be `playlist`, `track`, `album`, or `artist`. Defaults to `playlist` if not specified.
+
+### `limit` (int)
+
+*Optional*
+
+Sets a limit to the number of results to retrieve. Defaults to 10 if not specified.
+
+### `account` (str)
+
+*Optional*
+
+The `entry_id` of the account to use for Spotcast. If empty, the default Spotcast account is used.
+
+## Response
+```json
+{
+    "id": 7,
+    "type": "result",
+    "result": {
+        "total": 5,
+        "account": "01JDG07KSBTYWZGJSBJ1EW6XEF",
+        "playlists": [
+            {
+                "id": "37i9dQZF1DXcBWIGoYBM5M",
+                "name": "Rock Classics",
+                "uri": "spotify:playlist:37i9dQZF1DXcBWIGoYBM5M",
+                "description": "Classic rock hits",
+                "icon": "https://link_to_image.com"
+            },
+            {
+                "id": "37i9dQZF1DX4UtD7rx6U8w",
+                "name": "Indie Rock",
+                "uri": "spotify:playlist:37i9dQZF1DX4UtD7rx6U8w",
+                "description": "Indie rock tracks",
+                "icon": "https://link_to_image.com"
+            },
+            {...},
+            {...}
+        ]
+    }
+}
+```
+### `id` (int)
+
+The id provided in the request.
+
+### `type` (str)
+
+Always `result` on a successful request.
+
+### `result` (dict)
+
+The result of the transaction.
+
+#### `total` (int)
```

Comment:
Add quote style indentation for sub level definition see [raw account doc](https://raw.githubusercontent.com/mikevanes/spotcast/refs/heads/generic_get_ws/docs/websocket/accounts.md) for example